java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -425,8 +425,42 @@ class DatabaseImpl(
     override suspend fun delete(
         storageKey: StorageKey,
         originatingClientId: Int?
-    ): Unit = stats.delete.timeSuspending<Unit> {
-        TODO("not implemented")
+    ): Unit = stats.delete.timeSuspending { counters ->
```

Comment:
Are these changes supposed to be here? I don't think so...

Reply:
No, I pulled and rebased on my laptop and things got funky. Fixed it from my workstation this morning.